libsolidity/interface/CompilerStack.cpp

```diff
@@ -251,7 +251,10 @@ bool CompilerStack::parse()
 
 bool CompilerStack::analyze()
 {
-	if (m_stackState != ParsingSuccessful || m_stackState >= AnalysisSuccessful)
+	if (
+		(m_stackState != ParsingPerformed && !m_parserErrorRecovery) ||
```

Comment:
Why the additional check `&& !m_parserErrorrecovery`?

Reply:
Removed in 7fd7ad6fa